messenger.go

```diff
@@ -22,7 +22,7 @@ type Event struct {
 	Username  string   `json:"user"`
 	Filepath  string   `json:"filepath"`
 	Filesize  int64    `json:"filesize"`
-	Checksum  Checksum `json:"encoded_checksum"`
+	Checksum  Checksum `json:"encrypted_checksums"`
```

Comment:
This should be a list.

Reply:
`{
    "operation": "upload",
    "user": "username",
    "filepath": "/inbox/file",
    "filesize": 1234,
    "encrypted_checksums": [
        {
            "type": "md5",
            "value": "123qweasdzxc"
        }
    ]
}`